src/js/components/Layer/doc.js

```diff
@@ -156,6 +156,12 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
+  'layer.container.elevation': {
+    description: 'The amount of elevation. Any valid Box elevation value.',
```

Comment:
I don't think that 'The amount of elevation' is clear.
Can we use something like 'Elevated height above the underlying container, indicated via a drop shadow. Any Box elevation value is valid.'

Reply:
Yes, I like that wording. I'll update now.